Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -296,6 +349,21 @@ extension NavigationMapView {
             })
         }
         
+        private func calculateJaccardSimilarity(targetString: String, compareSet: Set<String>, allCounts: Int) -> Double {
+            let seperateStrings = targetString.components(separatedBy: " ")
+
+            var interSectionSize = 0
+            var unionSize = allCounts
+            for (_, string) in seperateStrings.enumerated() {
+                if compareSet.contains(string) {
+                    interSectionSize += 1
+                } else {
+                    unionSize += 1
+                }
+            }
+            return Double(interSectionSize) / Double(unionSize)
```

Comment:
I think whole algorithm can be simplified to something like:
```swift
let queriedRoadNames = Set<String>()
let electronicHorizonRoadNames = Set<String>()

let jaccardSimilarity =
    Double(queriedRoadNames.intersection(electronicHorizonRoadNames).count) /
    Double(queriedRoadNames.union(electronicHorizonRoadNames).count)
```

Reply:
Thanks for the suggestion. The `queriedRoadNames ` should be kept as an array of String instead of a set of String because of the following case. 
```
electronicHorizonRoadNames: ["Tom Tom Street"]
queriedRoadNames: ["Tom Tom St", "Tom Past St"]
```